examples/server/public/index.html

```diff
@@ -227,6 +227,15 @@ <h3 class="text-lg font-bold mb-6">Settings</h3>
             <summary class="collapse-title font-bold">Advanced config</summary>
             <div class="collapse-content">
               <label class="form-control mb-2">
+                <!-- Samplers queue -->
```

Comment:
I think this should be inside "Other sampler settings"

The "advanced" is mostly for non-sampler stuff I think, like pinning slot number or stop sequence

Reply:
btw if you merge with latest upstream master branch, you can re-use the `settings-modal-short-input` component

Reply:
> you can re-use the `settings-modal-short-input` component

Thanks, it works! However, the name is automatically set to `samplers`, which is not very descriptive.

Should I modify `settings-modal-short-input` to have a separate value for name?